index.html

```diff
@@ -29,26 +29,33 @@
         </div>
         <div class="hero">
             <nav>
-                <h2 class="logo"><span>R</span>.compaign</h2>
-                <ul>
-                    <li><a href="#">About</a></li>
+                <h2 class="logo"><a href=""><span>R</span>.company</a></h2>
+                <ul class=".nav-menu1">
+                    <li><a href="about.html">About</a></li>
                     <li><a href="#">Join</a></li>
                     <li><a href="#">Program </a></li>
                     <li><a href="#">Sponsor</a></li>
                     <li><a href="#">News</a></li>
-                    <button type="button">R.compaign</button>
+                    <button type="button">R.company</button>
                 </ul>
+                <button class="nav-icon" id="hamburger-btn">
+                    <i class="fas fa-bars"></i>
+                  </button>
+
+                  <button class="nav-icon" id="close-btn" >
+                    <i class="fas fa-times"></i>
+                  </button>
                 
             </nav>
```

Comment:
- [ ] Required
- [ ] Good Effort 👍 
- [ ] The top navbar icons are supposed to on the right as the original design.
- [ ] as well as remove all those boxes
- [ ] please make amends 🥺 

`Your template`
![image](https://github.com/RolandAmini/Capstone-project/assets/66230306/9b5edd1d-b3ad-4f40-b832-a406df1e493d)

`original`
![image](https://github.com/RolandAmini/Capstone-project/assets/66230306/f558993e-7369-4588-bd77-1e37b12c03cc)